composer/core/precision.py

```diff
@@ -8,8 +8,11 @@ class Precision(StringEnum):
     
     Attributes:
         AMP: Use :mod:`torch.cuda.amp`. Only compatible with GPUs.
+        FP16: Use 16-bit floating-point precision. Currently only
```

Comment:
is this BF16?

Reply:
No. BF16 assigns different amounts of bits to the mantissa and exponent. DeepSpeed does have some BF16 support but it's very limited right now.